packages/server/src/integration-test/postgres.spec.ts

```diff
@@ -89,7 +88,7 @@ describe("postgres integrations", () => {
     }
 
     primaryPostgresTable = await config.createTable({
-      name: `p_${generator.word({ length: 6 })}`,
+      name: `p_${generator.guid().replaceAll("-", "").substring(0, 6)}`,
```

Comment:
We should either use `word`, or create a custom helper to DRY these lines

Reply:
`word` doesn't provide enough uniqueness. I erred away from creating a helper because the line of code is simple enough to be understood in-place, while a helper would require you to go to its definition to understand what it's doing.

I don't mind creating a helper if that's what you'd prefer, but I don't personally consider the repetition here that much of a problem.